.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:   
+      release_tag:
+        description: 'release tag'
+        required: true
+      release_title:
+        description: 'release title'
+        required: false
+      release_notes:
+        description: 'release notes'     
+        required: false
+        default: ''         
+
+jobs:
+  create_release:
+    name: create_release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: prepare binaries for release artifact
+        run: |
+          mkdir -p binaries
+          declare -a ARCH=("amd64" "arm64")
+          for val in ${ARCH[@]}; do
+            mv custom/nsutil/nsutil-linux-${val} binaries/nsutil-linux-${val}
```

Comment:
We can identify an approach that can push binaries into release bundle but not maintain it locally. I guess we could take this improvement in another PR.

Reply:
Have made this change @ksatchit !

Now we are only keeping promeql cli binary in the repo and removed rest.